packages/cli/src/platform/_lib/utils.ts

```diff
@@ -29,6 +29,10 @@ export const platformParameters = {
     '--serviceToken': String,
     '-s': '--serviceToken',
   },
+  apikey: {
+    '--token': String,
+    '--apikey': String,
```

Comment:
I thought `--apikey` was a boolean?
Because if it's a value (and I guess a secret) then it must be added in
- https://github.com/prisma/prisma/blob/main/packages/cli/src/utils/checkpoint.ts#L155
- with an addition to the test here https://github.com/prisma/prisma/blob/main/packages/cli/src/__tests__/checkpoint.test.ts#L119

Reply:
Same for `--serviceToken`

Reply:
It is a resource ID, so not a secret.